CRM/Report/Form/Mailing/Bounce.php

```diff
@@ -470,7 +470,7 @@ public function alterDisplay(&$rows) {
       // If the email address has been deleted
       if (array_key_exists('civicrm_email_email', $row)) {
         if (empty($rows[$rowNum]['civicrm_email_email'])) {
-          $rows[$rowNum]['civicrm_email_email'] = '<del>Email address deleted</del>';
+          $rows[$rowNum]['civicrm_email_email'] = ts('<del>Email address deleted</del>');
```

Comment:
@yashodha i think for the two mailing ones here i think it might better if it was like this `'<del>' . ts('Email address deleted') . '</del>'` what do you think @mlutfy